crates/bitwarden/README.md

```diff
@@ -46,7 +47,8 @@ async fn test() -> Result<()> {
     let token = AccessTokenLoginRequest { access_token: String::from("") };
     client.access_token_login(&token).await.unwrap();
 
-    let org_id = SecretIdentifiersRequest { organization_id: String::from("") };
+    // The "v4" feature must be enabled to use `new_v4()` in the uuid crate
+    let org_id = SecretIdentifiersRequest { organization_id: Uuid::new_v4() };
```

Comment:
I think we should probably show an example parsing the UUID instead of generating a random one, like in `lib.rs`. I assume that's going to be the most common use case:

https://github.com/bitwarden/sdk/blob/729266454f48f9773802f714404892a614765269/crates/bitwarden/src/lib.rs#L49

To avoid having broken code in the Readme in the future, we can add something like this in the readme:
```rust
#[doc = include_str!("../README.md")]
mod readme {}

```

Reply:
Sounds good, and great idea. Updated 👍